packages/lesswrong/components/comments/CommentActions/CommentActions.tsx

```diff
@@ -54,7 +54,7 @@ const CommentActions = ({currentUser, comment, post, showEdit}: {
     {post && <ReportCommentMenuItem comment={comment}/>}
     {post && <MoveToAlignmentMenuItem comment={comment} post={postDetails}/>}
     {post && <SuggestAlignmentMenuItem comment={comment} post={postDetails}/>}
-    { userCanModeratePost(currentUser, postDetails) && postDetails?.user && userCanModeratePost(postDetails.user, postDetails) && <Divider />}
+    { userCanModeratePost(currentUser, postDetails) && postDetails?.user && <Divider />}
```

Comment:
This didn't typecheck (when type annotations more complete) because while we have enough fields to determine whether the current user can moderate this posts, we don't have the fields to determine whether the post's author can. But this was a really weird thing to be checking for in this context, so just, don't check for that. (The only thing it's gating is a divider-line in the menu.)

Reply:
Yeah, this will only affect admins, so should be fine.